app/views/layouts/application.html.erb

```diff
@@ -12,22 +12,16 @@
   <body>
     <%= render partial: "layouts/navbar" %>
     <%= yield %>
-    <%= render partial: "layouts/footer" %>
-    <% if flash[:notice] %>
-      <script>
-        toastr.options = {
-            "closeButton": true
-        };
-          toastr["success"]("<%= flash[:notice] %>")
+   <script>
+      toastr.options = {
+        "closeButton": true
+      };
+      <% if flash[:notice] %>
+        toastr["success"]("<%= flash[:notice] %>")
+      <% end %>
+      <% if flash[:alert] %>
+        toastr["warning"]("<%= flash[:alert] %>")
+      <% end %>
       </script>
```

Comment:
can we move this to a separate partial?

Reply:
this moved to a separate partial file @iMagesh